test/session.js

```diff
@@ -15,6 +15,8 @@ const sinon = require('sinon')
 tap.test('Create session', withServer((t, server) => {
   const configStub = sinon.stub(config, 'getConf')
   configStub.withArgs('authentication:type').returns('jwt')
+  configStub.withArgs('authentication:jwt').returns(null)
+  configStub.withArgs('authentication:secret').returns('test secret')
```

Comment:
Maybe we should update this test to make use of the new structure instead of the deprecated property
`jwt.secret`